src/main.rs

```diff
@@ -31,6 +31,10 @@ struct Args {
     /// Run as git commit hook.
     #[arg(long, value_delimiter = ' ', num_args = 1..3)]
     hook: Vec<String>,
+
+    /// Prints the local emoji cache path.
+    #[arg(short, long, default_value_t = false)]
+    print_emoji_cache_path: bool,
```

Comment:
We don't need the name to be so descriptive. The help message does that for us. `cache_path` is sufficient.

Reply:
That's the way I usually do: long, descriptive names. But I will change it, no worries.

Reply:
Yeah, me too but at the same time, not more than needed. E.g here it's obvious we're talking about `emoji` :)

Reply:
but cutting `print` will cause degradation in the readability.

Reply:
> but cutting print will cause degradation in the readability.

Don't think so since there are not that may possibilities here and behavior is the most obvious one.

Besides, people will likely find out this option through `--help` and that provides the description right next to it.

Anyway. Let's not bikeshed this to death. :laughing: If you still insist on `print-` prefix being important, do bring it back (but let's keep `-c` as the shortcut still).